Extension/tools/OptionsSchema.json

```diff
@@ -7,80 +7,80 @@
   "definitions": {
     "PipeTransport": {
       "type": "object",
-      "description": "When present, this tells the debugger to connect to a remote computer using another executable as a pipe that will relay standard input/output between VS Code and the MI-enabled debugger backend executable (such as gdb).",
+      "description": "%c_cpp.debuggers.pipeTransport.description%",
```

Comment:
Can we not have the ```%c_cpp.debuggers.*``` in this file? I would prefer that we have the original English text in the file. Have them write to the nls.json and then replace with the ```%..%``` when written to the package.json.

This schema file makes it easier for us to read all the flags we have for the debugger without the duplicates.

Reply:
To do what you're suggesting would require some mechanism to map English strings (or nodes) in OptionsSchema.json to the appropriate %% variables.  That seems much more complicated (and error prone) than using the %% vars and maintaining these strings in package.nls.json instead of OptionsSchema.json.

Reply:
Since this file isn't shipped but is used to generate the package.json anyways, could we have it generate a .nls.json file and then after the loc has happened, build the package.json?

Reply:
I'm open to a reasonable justification for doing this differently, but so far the complexity of what you suggesting does not seem a fair trade off for the sake of being able to keep English strings in this file.

Reply:
I'll make a separate PR after this is checked in to handle how we generate OptionsSchema.json and replace it with the resource strings.